tee-worker/litentry/pallets/identity-management/src/tests.rs

```diff
@@ -46,6 +46,13 @@ fn set_user_shielding_key_works() {
 #[test]
 fn create_identity_works() {
 	new_test_ext().execute_with(|| {
+		let shielding_key: UserShieldingKeyType = [0u8; USER_SHIELDING_KEY_LEN];
+		assert_ok!(IMT::set_user_shielding_key(
+			RuntimeOrigin::signed(ALICE),
```

Comment:
ALICE happens to work here because it's the same as the `ManagerOrigin` - maybe we should use either `One` here or modify the ManagerOrigin in `mock.rs` also to `ALICE`

Reply:
not applicable any more.